pkgs/applications/networking/seaweedfs/default.nix

```
@@ -15,6 +20,11 @@ buildGoModule rec {
 
   subPackages = [ "weed" ];
 
+  passthru.tests.check-version = runCommand "weed-version" {} ''
+    ${seaweedfs}/bin/weed version | grep -Fw ${version}
+    mkdir $out
```

```suggestion
    touch $out
```
i feel like a file is less resources than a directory, but the file might be in a directory, so it's 2 inodes instead of 1. not sure what makes more sense here and it probably don't make much difference. feel free to ignore